src/js/modules/product-subscription-abtest.js

```diff
@@ -0,0 +1,16 @@
+$('#subscriptionCheckbox').on('change', function () {
+	const subscriptionVariant = $('.product-variant--subscription');
+	subscriptionVariant.find('input[type="radio"]').prop('checked', true);
+	subscriptionVariant.find('label').toggleClass('bg-white', !this.checked);
+	subscriptionVariant.find('.product-variant__title').toggleClass('d-none', this.checked);
+	subscriptionVariant.find('.product-variant__title-subscription').toggleClass('d-none', !this.checked);
```

Comment:
all this can be achieved by adding only 1 class to the parent element and for children we can add styles in CSS by using that parent class.